Doc/library/dis.rst

```diff
@@ -1136,10 +1136,10 @@ All of the following opcodes use their arguments.
    .. versionadded:: 3.7
 
 
-.. opcode:: MAKE_FUNCTION (argc)
+.. opcode:: MAKE_FUNCTION (flags)
 
    Pushes a new function object on the stack.  From bottom to top, the consumed
-   stack must consist of values if the argument carries a specified flag value
+   stack must consist of values if *flags* carries a specified flag value
```

Comment:
I would keep this line as is.

Reply:
I don't have an idea if this should be called `operand` or `argument`, but we shall keep consistent in the whole Python documentations.
Just changed it according to your review. If you then think `argument` is not that proper just say something.